Cargo.toml

```diff
@@ -512,3 +514,6 @@ path = "src/bin/coreutils.rs"
 name = "uudoc"
 path = "src/bin/uudoc.rs"
 required-features = ["uudoc"]
+
+[package.metadata.cargo-udeps.ignore]
+normal = ["help_parser"]
```

Comment:
Why this?

Reply:
`cargo-udeps` complains otherwise about the `help_parser` dependency being unused.

Reply:
I figured this out. It's only a (runtime) dependency if the uudoc feature is enabled. So you need this under `[features]`:
```
uudoc = ["zip", "dep:help_parser"]
```
and
```
help_parser = { path="src/help_parser", optional=true }
```

Reply:
(If you remove "dep:" the build script will think `help_parser` is a util)